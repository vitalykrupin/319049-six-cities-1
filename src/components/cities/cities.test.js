import React from 'react';
import renderer from 'react-test-renderer';
import Cities from '../cities/cities';

describe(`Cities`, () => {
  const cities = [`Berlin`, `Dusseldorf`];

  it(`renders correctly`, () => {
    const tree = renderer
      .create(
          <Cities
            cities={cities}
            city={{name: `Berlin`}}
            onCityClick={jest.fn()}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
