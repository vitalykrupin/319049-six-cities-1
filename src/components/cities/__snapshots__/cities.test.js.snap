// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cities renders correctly 1`] = `
Array [
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item tabs__item--active"
      href="#"
    >
      <span>
        Berlin
      </span>
    </a>
  </li>,
  <li
    className="locations__item"
    onClick={[Function]}
  >
    <a
      className="locations__item-link tabs__item"
      href="#"
    >
      <span>
        Dusseldorf
      </span>
    </a>
  </li>,
]
`;
